client/src/templates/Introduction/components/block.tsx

```diff
@@ -118,21 +114,15 @@ class Block extends Component<BlockProps> {
     });
 
     const isProjectBlock = challenges.some(({ challenge }) => {
-      const isTakeHomeProject = blockDashedName === 'take-home-projects';
-
-      const projectCondition = [
-        isFinalProject(challenge.challengeType),
-        isCodeAllyPractice(challenge.challengeType)
-      ].some(Boolean);
-
-      return projectCondition && !isTakeHomeProject;
+      return isProjectBased.includes(challenge.challengeType);
     });
 
-    const isGridBlock = [
-      isNewRespCert(superBlock),
-      isNewJsCert(superBlock),
-      isSciCompPyCert(superBlock) && !isProjectBlock
-    ].some(Boolean);
+    const isGridBlock = challenges.some(({ challenge }) => {
+      return (
+        isGridBased.includes(superBlock) &&
+        !isMultipleProjectBased.includes(challenge.challengeType)
```

Comment:
A block with multiple projects would otherwise be shown as single certification project. This is what makes it necessary.